tests/test_keras_integration.py

```diff
@@ -178,7 +178,43 @@ def test_save_pretrained(self):
 
         model.build((None, 2))
 
-        save_pretrained_keras(model, f"{WORKING_REPO_DIR}/{REPO_NAME}")
+        save_pretrained_keras(
+            model,
+            f"{WORKING_REPO_DIR}/{REPO_NAME}",
+        )
+        files = os.listdir(f"{WORKING_REPO_DIR}/{REPO_NAME}")
+
+        self.assertIn("saved_model.pb", files)
+        self.assertIn("keras_metadata.pb", files)
+        self.assertEqual(len(files), 4)
+
+    def test_save_pretrained_optimizer_weights(self):
```

Comment:
nit: Should it be `optimizer_state` instead?

Reply:
there's no such thing. I load the model and check if `model.optimizer` is `None` or not

Reply:
will fix it